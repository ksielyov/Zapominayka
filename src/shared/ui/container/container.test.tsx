import renderer from 'react-test-renderer';
import {Container} from '@ui';
import {View} from 'react-native';
import {render} from '@testing-library/react-native';
import React from 'react';

test('container should render correctly', () => {
  const tree = renderer
    .create(
      <Container>
        <View />
      </Container>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('container should have view nested', () => {
  const testID = 'nestedView';

  const {queryByTestId} = render(
    <Container>
      <View testID={testID} />
    </Container>,
  );
  const element = queryByTestId(testID);
  expect(element).toBeTruthy();
});
