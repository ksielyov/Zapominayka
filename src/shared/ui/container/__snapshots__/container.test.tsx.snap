// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "paddingHorizontal": 34.285714285714285,
        "width": "100%",
      },
      Object {
        "marginTop": undefined,
        "position": undefined,
        "zIndex": undefined,
      },
    ]
  }
>
  <View />
</View>
`;
